docs/administrator.md

```diff
@@ -993,81 +993,7 @@ with `USE_WALG_BACKUP: "true"`.
 
 ### Google Cloud Platform setup
 
-When using GCP, there are two authentication methods to allow the postgres
-cluster to access buckets to write WAL-E logs: Workload Identity (recommended)
-or using a GCP Service Account Key (legacy).
-
-#### Workload Identity setup
```

Comment:
why is this all dropped?
I have no idea about GCP, but the PR seems to be about CRD change to allow to customize SecurityContext.
Please make sure the focus of the PR stays in line with the changes.